src/plugins/auto/src/plugin.cpp

```diff
@@ -455,8 +459,14 @@ std::shared_ptr<ov::ICompiledModel> Plugin::compile_model_impl(const std::string
         }
         LOG_INFO_TAG("device:%s, priority:%ld", iter->device_name.c_str(), iter->device_priority);
     }
-    auto_s_context->m_startup_fallback = load_config.get_property(ov::intel_auto::enable_startup_fallback);
-    auto_s_context->m_runtime_fallback = load_config.get_property(ov::intel_auto::enable_runtime_fallback);
+    auto_s_context->m_startup_fallback =
+        full_property.count(ov::intel_auto::enable_startup_fallback.name())
```

Comment:
can we optimize this part? it's confused to check a flag from two property list

Reply:
updated.